examples/docker-compose/docker-compose.yaml

```diff
@@ -66,34 +66,32 @@ services:
       - "traefik.http.routers.auth.middlewares=strip-auth@docker"
       - "traefik.http.routers.auth.entrypoints=web"
   storage:
-    image: nhost/hasura-storage:latest
+    image: nhost/hasura-storage:0.1.5
```

Comment:
`nhost/hasura-storage:0.1.5` is not available on Docker Hub. Is it about to be released?